Tutorials/intro-to-qdac2/Integration_with_the_OPX_and_pyvisa.ipynb

```diff
@@ -450,11 +471,34 @@
     "qdac.write(f\"sour1:volt {0}\")\n",
     "qdac.write(f\"sour2:volt {0}\")"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Example:\n",
+    "1. Setting the QDAC on a specific voltage - in the sexample you can set the QDAC on a specific, fixed, value. "
```

Comment:
example instead of sexample
maybe: "1. Setting the QDAC to output a specific constant voltage."